releasenotes/notes/observable-eval-primitives-e1fd989e15c7760c.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    Added `~qiskit.algorithms.observables_evaluator` with
```

Comment:
This should be qualified with :meth: I think and maybe we should show the full path for enough context so no ~